website/project/metadata/authorizers/__init__.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+
+groups = json.load(open('{0}/defaults.json'.format(HERE)))
+fp = None
+try:
+    fp = open('{0}/local.json'.format(HERE))
```

Comment:
This filepointer and the one to `defaults.json` never get closed. `with open(...)` should be used.